### PR TITLE
Add camera timestamp override option

### PR DESCRIPTION
**Pull request by KhushrajRathod**

### CODE REVIEW

File: app/src/main/java/me/phh/treble/app/Misc.kt [hunk 1]
```diff
@@ -44,10 +44,15 @@ object Misc: EntryStartup {
                 Log.d("PHH", "Setting signal level method to $value")
             }
             MiscSettings.fpsDivisor -> {
-                val value = sp.getString(key, "1")
```
Comment (phhusson) [inline app/src/main/java/me/phh/treble/app/Misc.kt:47]: What

File: app/src/main/res/xml/pref_misc.xml [hunk 2]
```diff
@@ -59,6 +59,23 @@
...
+		<CheckBoxPreference
+						android:defaultValue="false"
+						android:key="key_misc_multi_camera"
+						android:title="Expose Aux cameras (Qualcomm)" />
+		<CheckBoxPreference
+				android:defaultValue="false"
+				android:key="key_misc_force_camera2api_hal3"
+				android:title="Force Enable Camera2API HAL3" />
+		<ListPreference
+		    android:defaultValue="1"
```
Comment (phhusson) [inline app/src/main/res/xml/pref_misc.xml:72]: -1

File: app/src/main/java/me/phh/treble/app/Misc.kt (hunk 1 above)
Comment (KhushrajRathod) [reply app/src/main/java/me/phh/treble/app/Misc.kt]: Ugh I seriously don't know how I manage to do such things... Changed

File: app/src/main/res/xml/pref_misc.xml (hunk 2 above)
Comment (KhushrajRathod) [reply app/src/main/res/xml/pref_misc.xml]: Done

**Pull request merged**